### PR TITLE
feat: feemarket module

### PR DESCRIPTION
**Pull request by nowooj**

### CODE REVIEW

File: content/en/docs/develop/core-modules/feemarket.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+```
+
+where `gasPrice` is the price per gas and `gasLimit` describes the amount of gas required to perform the transaction.
+The more complex operations a transaction requires, the higher the gasLimit (See [Executing EVM bytecode](https://docs.evmos.org/protocol/modules/evm#concepts)).
+To submit a transaction, the signer needs to specify the `gasPrice`.
+
+{{< alert >}}
+The Cosmos SDK uses a different terminology for `gas` than Ethereum.
+What is called `gasLimit` on Ethereum is called `gasWanted` on Cosmos.
+You might encounter both terminologies on Evmos since it builds Ethereum on top of the SDK,
```
Comment (jhlee-young) [inline content/en/docs/develop/core-modules/feemarket.md]: plz update chain name 👀

Comment (nowooj) [reply content/en/docs/develop/core-modules/feemarket.md]: https://github.com/xpladev/docs/pull/15/commits/52cdf4abe1d6f96b1b36e25a11f02f819dbd82c5

**Review by jhlee-young — APPROVED** [review]
LGTM

**Review by caramis — APPROVED** [review]
LGTM

**Pull request merged**